frontend/package.json

```diff
@@ -15,26 +15,25 @@
     "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.3.0",
-    "@fortawesome/free-regular-svg-icons": "6.1.1",
-    "@fortawesome/free-solid-svg-icons": "6.1.1",
-    "@fortawesome/vue-fontawesome": "^3.0.1",
-    "@rollup/plugin-strip": "^2.1.0",
-    "@vueform/multiselect": "^2.3.1",
-    "vue": "3.2.25",
-    "vue-router": "4"
+    "@fortawesome/fontawesome-svg-core": "6.2.1",
+    "@fortawesome/free-regular-svg-icons": "6.2.1",
+    "@fortawesome/free-solid-svg-icons": "6.2.1",
+    "@fortawesome/vue-fontawesome": "3.0.2",
+    "@rollup/plugin-strip": "3.0.2",
+    "vue": "3.2.45",
+    "vue-router": "4.1.6"
   },
   "devDependencies": {
-    "@vitejs/plugin-vue": "2.0.1",
-    "@vitest/ui": "0.10.0",
-    "@vue/test-utils": "2.0.0-rc.18",
-    "c8": "^7.11.3",
-    "eslint": "^8.7.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-plugin-vue": "^8.4.0",
-    "happy-dom": "^2.25.1",
-    "sinon": "^13.0.1",
-    "vite": "2.9.6",
-    "vitest": "0.10.0"
+    "@vitejs/plugin-vue": "4.0.0",
+    "@vitest/coverage-c8": "0.27.2",
+    "@vitest/ui": "0.27.2",
+    "@vue/test-utils": "2.2.7",
+    "eslint": "8.32.0",
+    "eslint-config-google": "0.14.0",
+    "eslint-plugin-vue": "9.9.0",
+    "happy-dom": "6.0.4",
+    "sinon": "15.0.1",
+    "vite": "4.0.4",
+    "vitest": "0.27.2"
```

Comment:
> yarn test:unit

**The tests pass:**

![Screenshot 2023-01-31 at 1 06 50 PM](https://user-images.githubusercontent.com/5799712/215858004-6f4e4fcb-28e4-494d-bb24-1b9768f5a722.png)

**But they have a bunch of extra warnings:**

![Screenshot 2023-01-31 at 1 04 38 PM](https://user-images.githubusercontent.com/5799712/215857878-044be860-c94d-461d-ac77-fa9b4d66336b.png)

Reply:
This is now passing!

<img width="893" alt="Screenshot 2023-02-01 at 10 24 43 AM" src="https://user-images.githubusercontent.com/5799712/216102252-92ad8f02-0d09-4cf1-b737-9b661233cbe7.png">